flyctl/app_config_test.go

```diff
@@ -54,3 +54,21 @@ func TestLoadTOMLAppConfigWithServices(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, p.Definition, rawData)
 }
+
+func TestLoadTOMLAppConfigWithEnvVars(t *testing.T) {
+	path := "./testdata/env-vars.toml"
```

Comment:
Maybe this should be a constant `const path = "testdata/env-vars.toml"`.

Reply:
Do you think I should standardise all the rest of the test as well? seems like a good idea.